extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/DurableStorageCleanerConfig.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.msq.indexing;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Preconditions;
+
+public class DurableStorageCleanerConfig
+{
+
+  private static final long DEFAULT_INITIAL_DELAY_SECONDS = 86400L;
+  private static final long DEFAULT_DELAY_SECONDS = 86400L;
+
+  /**
+   * Whether the {@link DurableStorageCleaner} helper should be enabled or not
+   */
+  @JsonProperty
+  private final boolean enabled;
+
+  /**
+   * Initial delay in seconds post which the durable storage cleaner should run
+   */
+  @JsonProperty
+  private final long initialDelaySeconds;
+
+  /**
+   * The delay (in seconds) after the last run post which the durable storage cleaner would clean the outputs
+   */
+  @JsonProperty
+  private final long delaySeconds;
+
+  @JsonCreator
+  public DurableStorageCleanerConfig(
+      @JsonProperty("enabled") final Boolean enabled,
+      @JsonProperty("initialDelay") final Long initialDelaySeconds,
+      @JsonProperty("delay") final Long delaySeconds
+  )
+  {
+    this.enabled = enabled != null && enabled;
+    this.initialDelaySeconds = initialDelaySeconds != null ? initialDelaySeconds : DEFAULT_INITIAL_DELAY_SECONDS;
```

Comment:
Nit: I would be curious to understand why is an InitialDelay required. Maybe I am missing a use case?

Reply:
I added the `initialDelay` based off the existing TaskLogCleaner's configs. The use case that I thought of was that setting it to a lower value (or 0) for the initial run would ensure that the durable storage is cleaned as soon as the helper is scheduled for an initial cleanup.
If it is adding complexity, I can remove this and set the initial cleanup to either `0` or `delaySeconds`. WDYT?

Reply:
I think lets keep it simple and remove it. The initial delay can be `delaySeconds`